### ISSUE TITLE
Option on_close: :stop_viewport doesn't work - doesn't close the viewport

### ISSUE THREAD

**Issue by kelostrada** [boydm/scenic#166]
### Versions and Environment

Elixir: 1.9.0

```bash
# elixir -v
Erlang/OTP 22 [erts-10.4.3] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [hipe] [dtrace]

Elixir 1.9.0 (compiled with Erlang/OTP 22)
```

Erlang:

```bash
# erl -v
Erlang/OTP 22 [erts-10.4.3] [source] [64-bit] [smp:12:12] [ds:12:12:10] [async-threads:1] [hipe] [dtrace]

Eshell V10.4.3  (abort with ^G)
```

Scenic:

```text
# mix deps | grep scenic
* scenic (Hex package) (mix)
  locked at 0.10.2 (scenic) f4f0a00e
* scenic_driver_glfw (Hex package) (mix)
  locked at 0.10.0 (scenic_driver_glfw) 01a00c8d
```

### Steps to reproduce

Set config option
```
on_close: :stop_viewport
```

### Expected Behavior

The viewport should be closed

### Actual Behavior

Nothing happens

### Additional Comments

I tried to track what happens and it seems that the dynamic supervisor which tries to close viewport isn't linked with it. There is a function `start` which seems to be unused and the code there seems to have something in common with the `terminate_child` function.

**Comment by boydm** [comment]
This sounds like a bug. I'm going to be digging around in Scenic again in a week or so and will take a look then. If you can further resolve any info on it, that would be appreciated.

**Comment by boydm** [comment]
This should be fixed in v0.11, which I'm about to merge into master
